src/Item.h

```diff
@@ -206,6 +222,7 @@ class cItem
 
 	int            m_RepairCost;
 	cFireworkItem  m_FireworkItem;
+	unsigned int m_ItemColor;
```

Comment:
I just don't think it should be it cItem, because we will add more complex logic involving colors.